dm/syncer/dbconn/db.go

```diff
@@ -202,8 +203,11 @@ func (conn *DBConn) ExecuteSQLWithIgnore(
 		})
 	if err != nil {
 		tctx.L().ErrorFilterContextCanceled("execute statements failed after retry",
+			zap.Int("numRetries", numRetries),
+			zap.Float64("costSeconds", time.Since(startTime).Seconds()),
 			zap.String("queries", utils.TruncateInterface(queries, -1)),
 			log.ZapRedactString("arguments", utils.TruncateInterface(args, -1)),
+			zap.Error(err),
 			log.ShortError(err))
```

Comment:
`zap.Error(err)` and `log.ShortError(err)` is a bit duplicated. Do you want to remove `log.ShortError(err)`?